### PR TITLE
Update API to not rely on deprecated feature.

### PR DESCRIPTION
**Pull request by Carreau**

### CODE REVIEW

**Review by alisterburt — APPROVED** [review]
I'm not 100% on the context here but the changes all look good, love the simplification of the actions configs - I'm happy to see this merged and happy with bumping the minimum version to 3.8 in line with NEP29

File: napari_console/qt_console.py [hunk 1]
```diff
@@ -161,9 +161,7 @@ def _update_theme(self, event=None):
         # (should probably be done by napari)
         # After napari 0.4.11, themes are evented models rather than
         # dicts.
-        with warnings.catch_warnings():
-            warnings.simplefilter("ignore", FutureWarning)
-            theme = get_theme(self.viewer.theme)
+        theme = get_theme(self.viewer.theme, as_dict=True)
```
Comment (Carreau) [inline napari_console/qt_console.py:164]: This is the reason for this full PR, `theme = get_theme(self.viewer.theme)` is deprecated, and replaced by `get_theme(self.viewer.theme, as_dict=True)`, once this is done we can remove the deprecated API from napri.

**Comment by Carreau** [comment]
> I'm not 100% on the context here

I commented on the relevant line that uses a deprecated API. 
I don't have any tight to merge anything here.

**Comment by alisterburt** [comment]
interesting, me neither... @napari/core-devs does anyone have write access here? If so could you also give me write access for the future? 🙂

**Comment by alisterburt** [comment]
Thanks @Carreau and whoever gave me write access!

**Pull request merged**